package.json

```diff
@@ -17,12 +17,14 @@
   "scripts": {
     "postinstall": "lerna bootstrap",
     "build": "lerna run compile",
-    "lint": "eslint . --ext .ts --ext .js && lerna run --scope @pipcook/pipcook-board-client lint",
+    "lint": "eslint . --ext .ts --ext .js --fix",
```

Comment:
It's not a good practice to run `--fix`, this causes CI is green even though some lint errors which could be fixed.